dartagnan/src/main/java/com/dat3m/dartagnan/verification/model/ExecutionModelManager.java

```diff
@@ -152,7 +148,9 @@ private void extractEvent(Event e, ThreadModel tm, int id) {
         } else if (e instanceof GenericVisibleEvent visible) {
             em = new GenericVisibleEventModel(visible, tm, id);
         } else if (e instanceof Assert assrt) {
-            em = new AssertModel(assrt, tm, id);
+            em = new AssertModel(
+                assrt, tm, id, isTrue(context.encodeExpressionAsBooleanAt(assrt.getExpression(), assrt))
```

Comment:
`assrt` -> `assert`

Reply:
`assert` as a keyword will confuse the compiler.